starter-core/src/test/groovy/io/micronaut/starter/feature/config/YamlSpec.groovy

```diff
@@ -21,6 +22,12 @@ class YamlSpec extends BeanContextSpec implements CommandOutputFixture {
         yaml.order == FeaturePhase.HIGHEST.getOrder()
     }
 
+    @PendingFeature
+    void "is visible"() {
+        expect:
+        yaml.isVisible()
+    }
+
```

Comment:
Is this to check it's not visible?

Or is it going to be visible one day?

If it's to check that `isVisible()` is false, and it's staying that way, we should remove `@PendingFeature` and just test that it is false

Reply:
This PR is for 3.9.0 and not visible. For 4.0.0 it will change to be visible, when the `snakeyaml` dependency will be added.

Reply:
👍